client/js/components/forms/LoginForm.js

```diff
@@ -25,23 +24,21 @@ class LoginForm extends Component {
   }
 
   render() {
-
     const { errors } = this.props;
 
     return (
-      <div>
-        <ErrorList errors={errors} showFieldErrors={true}/>
-        <form id="loginForm" onSubmit={this.onSubmit.bind(this)}>
-          <div className="row">
-            <Input type="email" placeholder="Email" addonBefore="@" onChange={this.onStateChange.bind(this, 'email')} required/>
-          </div>
-          <div className="row">
-            <Input type="password" placeholder="Password" onChange={this.onStateChange.bind(this, 'password')} required/>
-          </div>
-          <div className="row">
-            <Button className="col-md-4 col-md-push-4 btn btn-success btn-sm" type="submit">Login</Button>
-          </div>
-        </form>
+      <div id="loginForm">
+        <article>
+          <section>
+            <h2>Login</h2>
+            <ErrorList errors={ errors } showFieldErrors={ true } />
+            <form onSubmit={ this.onSubmit.bind(this) }>
```

Comment:
I dont have any prefered way for display erros messages. I think it's good above the form. usualy I just include a `error` class in the input with error.